apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -2,13 +2,14 @@ import { html } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-checkbox';
-import { CheckboxProps, defaultProps, statusTypes } from '@justeattakeaway/pie-checkbox';
+import { CheckboxProps as CheckboxBaseProps, defaultProps, statusTypes } from '@justeattakeaway/pie-checkbox';
```

Comment:
```suggestion
import { type CheckboxProps as CheckboxBaseProps, defaultProps, statusTypes } from '@justeattakeaway/pie-checkbox';
```

Reply:
I think?

Reply:
I noticed that we don't use `type` in any of other component stories (tag, notification, link and assistive text). Do we need to add it to all of them? Could you explain me why we need this change since I am a typescript noob.

Reply:
@dandel10n I have a PR (#1622) which adds a new linting rule and also adds it to the other stories, so it would be good if you could add it here please 😄

Adding `type` when an import is only used for TypeScript tells `tsc` to remove the import for the compiled JS, which in theory makes for smaller bundles. It's less important for storybook files but I think it's a good habit to get into.